__tests__/app.test.js

```diff
@@ -67,9 +66,17 @@ describe("GET /api/articles/:article_id", () => {
         );
       });
   });
+  it("should return an article object with the correct comment count", () => {
+    return request(app)
+      .get("/api/articles/3")
+      .expect(200)
+      .then((response) => {
+        const { body } = response;
+        expect(Number.parseInt(body.article.comment_count)).toBe(2);
+      });
+  });
```

Comment:
This parseInt of the count should be handled as part of the SQL query rather than in your tests.